external_libs.BUILD.bazel

```diff
@@ -44,7 +44,7 @@ cc_library(
         "mysql/typelib.h",
     ],
     linkopts = [
-        "-l:libmariadbclient.a",
+        "-l:/opt/mysql/10.1.37/lib/libmysqlclient.a",
```

Comment:
This is not related (and doesn't exist on my machine ;) ).

Reply:
Sorry for that, I should have been more careful.